### ISSUE TITLE
ExtractSemanticDB cant find output file in CommentPicklingTest on JDK 11

### ISSUE THREAD

**Issue by bishabosha** [lampepfl/dotty#9026]
## Minimized code

on adoptopenjdk 11.0.7
```Scala
> testOnly dotty.tools.dotc.core.tasty.CommentPicklingTest
```

## Output (click arrow to expand)
<details>

```scala
Test dotty.tools.dotc.core.tasty.CommentPicklingTest.commentOnClass failed: java.nio.file.NoSuchFileException: /tmp/temp9165565774282302089/out/META-INF/semanticdb/../../../tmp/temp9165565774282302089/Src$id.scala.semanticdb, took 0.04 sec
    at sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
    at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
    at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
    at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
    at java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
    at java.nio.file.Files.newOutputStream(Files.java:219)
    at dotty.tools.dotc.semanticdb.ExtractSemanticDB$.write(ExtractSemanticDB.scala:622)
    at dotty.tools.dotc.semanticdb.ExtractSemanticDB.run(ExtractSemanticDB.scala:46)
    at dotty.tools.dotc.core.Phases$Phase.runOn$$anonfun$1(Phases.scala:318)
    at scala.collection.immutable.List.map(List.scala:246)
    at dotty.tools.dotc.core.Phases$Phase.runOn(Phases.scala:319)
    at dotty.tools.dotc.Run.runPhases$4$$anonfun$4(Run.scala:165)
    at dotty.runtime.function.JProcedure1.apply(JProcedure1.java:15)
    at dotty.runtime.function.JProcedure1.apply(JProcedure1.java:10)
    at scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1323)
    at dotty.tools.dotc.Run.runPhases$5(Run.scala:175)
    at dotty.tools.dotc.Run.compileUnits$$anonfun$1(Run.scala:183)
    at dotty.runtime.function.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
    at dotty.tools.dotc.util.Stats$.maybeMonitored(Stats.scala:64)
    at dotty.tools.dotc.Run.compileUnits(Run.scala:190)
    at dotty.tools.dotc.Run.compileSources(Run.scala:127)
    at dotty.tools.dotc.Run.compile(Run.scala:110)
    at dotty.tools.dotc.Driver.doCompile(Driver.scala:38)
    at dotty.tools.dotc.Driver.process(Driver.scala:194)
...
```
</details>

**Comment by bishabosha** [comment]
it is not the cause but I found an un-interpolated string in that test https://github.com/lampepfl/dotty/blob/94494ff34c0c7e0a5b1f388d6cfabdd604a02ee1/compiler/test/dotty/tools/dotc/core/tasty/CommentPicklingTest.scala#L85

**Comment by bishabosha** [comment]
closed by #9027
